modules/javafx.graphics/src/main/java/javafx/stage/Stage.java

```diff
@@ -1183,6 +1183,8 @@ public void toBack() {
     /**
      * Closes this {@code Stage}.
      * This call is equivalent to {@code hide()}.
+     * @throws IllegalStateException if this method is called on a thread
+     * other than the JavaFX Application Thread.
```

Comment:
Minor: add a blank line before `@throws`. I also think it's easier to read docs when multi-line text for a javadoc tag is indented, either by four spaces or lined up with the beginning of the first line of text (i.e. with the beginning of `IllegalStateException`).

Reply:
good idea, even though it does not affect generated javadoc.
will do next time, to avoid re-setting the approvals ;-)

Reply:
> will do next time, to avoid re-setting the approvals ;-)

I'll re-approve if you decide to do the changes anyway.

Reply:
it is not required, and the code is inconsistent (Stage:472) ... but ok.

Reply:
Can you do it for the other changed docs also?

Reply:
You've added the same javadoc tag in 20 different places. All I'm asking is that you do the change that you did in this particular place in all of the other places too.

Reply:
I misunderstood, sorry.  Done.